argoverse/evaluation/detection_utils.py

```diff
@@ -163,6 +161,24 @@ def dist_fn(dts: pd.DataFrame, gts: pd.DataFrame, metric: DistFnType) -> np.ndar
         raise NotImplemented("This distance metric is not implemented!")
 
 
+def iou_aligned_3d(dt_dims: pd.DataFrame, gt_dims: pd.DataFrame) -> np.ndarray:
+    """Calculate the 3d, axis-aligned (vertical axis alignment) intersection-over-union (IoU)
+    between the detections and the ground truth labels after aligning their poses.
```

Comment:
i might state that the +x axis of both objects is aligned and the centroids are placed at the same location. "aligning their poses" is a bit more ambiguous to me